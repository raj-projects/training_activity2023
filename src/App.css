.App {
  text-align: center;
  margin: 0 auto;
}

.counter-section {
  margin: 0 auto;
  text-align: center;
  width: 200px;
  height: 200px;
  border: 1px solid#666;
}

.multi-counter {
  display: flex;
  justify-content: space-evenly;
}
