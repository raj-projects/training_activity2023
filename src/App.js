
import './App.css';

function App() {
  return (
    <div className="App">
      <h1>Unit Testing with React testing library</h1>
    </div>
  );
}

export default App;
