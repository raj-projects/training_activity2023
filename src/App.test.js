import { render, screen } from '@testing-library/react';
import App from './App';

test('renders Todo 1', () => {
  render(<App />);
  const linkElement = screen.getByText(/Todo 1/i);
  expect(linkElement).toBeInTheDocument();
});

test('renders Todo 2', () => {
  render(<App />);
  const linkElement = screen.getByText(/Todo 2/i);
  expect(linkElement).toBeInTheDocument();
});