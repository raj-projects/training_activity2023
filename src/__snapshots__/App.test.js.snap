// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      Unit Testing with React testing library
    </h1>
  </div>
</DocumentFragment>
`;
